pyproject.toml

```diff
@@ -23,11 +29,66 @@ docs = [
 default-groups = ["dev", "docs"]
 
 [tool.uv.workspace]
-members = ["stapi-pydantic"]
+members = ["stapi-pydantic", "stapi-fastapi"]
 
 [tool.uv.sources]
 stapi-pydantic.workspace = true
+stapi-fastapi.workspace = true
+
+[tool.ruff]
+line-length = 88
+include = [
+    "stapi-pydantic/src/stapi_pydantic/**/*.py",
+    "stapi-fastapi/src/stapi_fastapi/**/*.py"
+]
+
+[tool.ruff.format]
+quote-style = 'double'
+
+[tool.ruff.lint]
+extend-ignore = ["E501", "UP007", "UP034"]
+select = [
+    "C9",
+    "E",
+    "F",
+    "I",
+    "W",
+    "UP"
+]
+
+[tool.ruff.lint.mccabe]
+max-complexity = 8
 
 [tool.mypy]
 strict = true
-files = "stapi-pydantic/src/stapi_pydantic/**/*.py"
+files = [
+    "stapi-pydantic/src/stapi_pydantic/**/*.py",
+    "stapi-fastapi/src/stapi_fastapi/**/*.py"
+]
+
+[[tool.mypy.overrides]]
+module = "pygeofilter.parsers.*"
+ignore_missing_imports = true
+
+[tool.pymarkdown]
+plugins.md013.line_length = 120
+plugins.md024.enabled = false # duplicate headers
```

Comment:
Do we have duplicate headers to disable this for?

Reply:
changelog headers (Added, Changed, Fixed etc.). Adding ignore statements to each header doesn't look very clean https://pymarkdown.readthedocs.io/en/stable/advanced_plugins/#suppressing-rule-failures